### ISSUE TITLE
snuba keep crashing / restarting without any error message

### ISSUE THREAD

**Issue by gizmo84** [getsentry/self-hosted#2974]
### Self-Hosted Version

24.4.1 (but since 24.3.0)

### CPU Architecture

x86_64

### Docker Version

25.0.5

### Docker Compose Version

2.26.1

### Steps to Reproduce

Start Sentry, generate issue. No issues are displayed in the gui.

### Expected Result

See isssues again in the GUI

### Actual Result

Initializing Snuba...
Snuba initialization took 8.68588224000996s
Initializing Snuba...

**Comment by gizmo84** [comment]
We started a fresh Install on an other server, same result: even with a basic setup, Sentry doesn't display any "Issues" and has the same Problem with Snuba as our old instance:

Initializing Snuba...
Snuba initialization took 8.68588224000996s
Initializing Snuba...
...

**Comment by azaslavsky** [comment]
My first guess here would be a memory issue. Can you check the memory usage on this container, and the system asa whole, relative to how much you have available?

**Comment by gizmo84** [comment]
@azaslavsky I have 16 GB RAM on both VMs and it was working smoothly until 24.4.0

**Comment by azaslavsky** [comment]
I don't doubt that your setup is sufficient, but if the issue is related to memory consumption on our end due to something we snuck into the 24.4.0 release, it would be a helpful data point to know.

**Comment by gizmo84** [comment]
So I increased the RAM from 16 Gb to 24 Gb and the number of core on the machine. Snuba is still crashing.

I noticed one error in the log just befor the next "Initializing Snuba..."

{"timestamp":"2024-04-24T14:14:30.849469Z","level":"ERROR","fields":{"error":"the strategy errored"},"target":"rust_snuba::consumer"}

**Comment by azaslavsky** [comment]
Can you try the advice postede [here](https://github.com/getsentry/self-hosted/issues/2931#issuecomment-2040854990)?

**Comment by gizmo84** [comment]
Yes!! Sentry is working again for me. I replaced all occurrences of rust-consumer in docker-compose.yml file to just consumer.

Should I keep this issue open until it is fixed ?

**Comment by azaslavsky** [comment]
cc @volokluev. We'll keep it open for now.

**Comment by gizmo84** [comment]
I just upgraded from 24.5.0 to 24.8.0 and this is still an issue. Replaced again all occurrences of rust-consumer in docker-compose.yml file to just consumer...

**Comment by naanlizard** [comment]
How is this still an issue. This project is great but making the self hosted version fall over constantly is not the way to get people to want to use the SaaS version

**Comment by aldy505** [comment]
Since the issue lays on rust-consumers, let's just keep the issue here https://github.com/getsentry/self-hosted/issues/2931
